### PR TITLE
Added Issue details view

### PR DESCRIPTION
**Pull request by dilantha111**
when an issue is selected, issue details are showed in right side.

### CODE REVIEW

File: src/HtmlProvider.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const axios = require('axios')
+
+export class HtmlProvider{
+
+    private async _getIssues(url:string){
+        let result = []
```
Comment (rehrumesh) [inline src/HtmlProvider.ts]: Please add semicolon at the end. Applies to every other place as well

File: src/HtmlProvider.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        try{
+            let data = await axios.get(url)
+            result = data.data
+        }catch(e){
+            console.log(e)
```
Comment (rehrumesh) [inline src/HtmlProvider.ts]: Better to use `console.error` instead, don't you think?

File: src/HtmlProvider.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            output += '<ul>' + comments.join(' ') + '</ul>'
+            output += 
+            `<style>
+                img{
+                    max-width: 30px;
+                    max-height: 30px;
+                }    
+            </style>`
+        }catch(e){
+            output += 'Loading failed.. check your internet connection'
```
Comment (rehrumesh) [inline src/HtmlProvider.ts]: Sure this is only related to internet connection? Aren't there any other ways which could occur this error?

File: src/HtmlProvider.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            console.log(e)
```
Comment (rehrumesh) [inline src/HtmlProvider.ts]: `console.error` ??

File: src/HtmlProvider.ts (hunk 3 above)
Comment (dilantha111) [reply src/HtmlProvider.ts]: I changed it 'output += 'Loading failed.. '+e.message;' would that be suitable ?

**Review by rehrumesh — APPROVED** [review]

File: src/HtmlProvider.ts (hunk 3 above)
Comment (rehrumesh) [reply src/HtmlProvider.ts]: That is much better

**Pull request merged**